src/components/InventoryGroupDetail/GroupTabDetails.js

```diff
@@ -6,13 +6,14 @@ import {
   TabTitleText,
   Tabs,
 } from '@patternfly/react-core';
-import React, { Suspense, lazy, useState } from 'react';
+import React, { Suspense, lazy, useEffect, useState } from 'react';
 import { hybridInventoryTabKeys } from '../../Utilities/constants';
 import GroupSystems from '../GroupSystems';
```

Comment:
It feels like you are importing the wrong component: did you mean
```suggestion
import GroupSystems from '../GroupSystems/GroupSystems';
```
instead? The reason is that you later use the imported `GroupSystems` in the `conventionalSystemsContent` variable. But the current import that you have will render `GroupSystemsWrapper` (https://github.com/RedHatInsights/insights-inventory-frontend/pull/2102/files#diff-272350e154e0572ba97a0176c9495f955365b3b500f6e5c752f8c8717d0f20d0R9) component which does another check/condition. It seems to be a duplicating/double-work: if user clicks on the Conventional tab, then Conventional GroupSystems should be rendered.